dynatrace-oneagent-operator/README.md

```diff
@@ -0,0 +1,85 @@
+# Dynatrace OneAgent Operator Helm Chart
+
+This is the home of the Dynatrace OneAgent Operator's Helm Chart which supports the rollout and lifecycle of [Dynatrace OneAgent](https://www.dynatrace.com/support/help/get-started/introduction/what-is-oneagent/) in Kubernetes and OpenShift clusters.
+Rolling out Dynatrace OneAgent via DaemonSet on a cluster is straightforward.
+Maintaining its lifecycle places a burden on the operational team.
+Dynatrace OneAgent Operator closes this gap by automating the repetitive steps involved in keeping Dynatrace OneAgent at its latest desired version.
+
+## Overview
+
+Dynatrace OneAgent Operator is based on [Operator SDK](https://github.com/operator-framework/operator-sdk) and uses its framework for interacting with Kubernetes and OpenShift environments.
+It watches custom resources `OneAgent` and monitors the desired state constantly.
+The rollout of the Dynatrace OneAgent is managed by a DaemonSet initially.
+From here on the Dynatrace OneAgent Operator controls the lifecycle and keeps track of new versions and triggers updates if required.
+The Dynatrace OneAgent Operator's Helm Chart will also roll out the actual OneAgent automatically during installation.
+
+![Overview](./overview.svg)
+
+
+## Supported platforms
+
+Depending of the version of the Dynatrace OneAgent Operator, it supports the following platforms:
+
+| Dynatrace OneAgent Operator Helm Chart version | Kubernetes | OpenShift Container Platform |
+| ---------------------------------------------- | ---------- | ---------------------------- |
+| v0.5.4                                         | 1.11+      | 3.11+                        |
+
+
+## Quick Start
```

Comment:
Would add instructions to replace the API URL and tokens from the `helm install` commands below.